go/flags/endtoend/vtctld.txt

```diff
@@ -68,6 +68,7 @@ Usage of vtctld:
       --log_err_stacks                                                   log stack traces for errors
       --log_rotate_max_size uint                                         size in bytes at which logs are rotated (glog.MaxSize) (default 1887436800)
       --logtostderr                                                      log to standard error instead of files
+      --manifest-external-decompressor string                            command with arguments to store in the backup manifest when compressing a backup with an external compression engine.
```

Comment:
Why do we need to pass _any_ compression flags to vtctld? vttablet / vtbackup should be sufficient.
@rsajwani we may need to remove older compression related flags too.

Reply:
I don't need to! I'm just doing that because CI fails if I don't, because there's pre-existing code that registers `mysqlctl` flags with vtctld.

Do you want me to address this in this PR, or leave it to a separate PR to tackle that?

Reply:
@maxenglander , I would recommend to do it in this PR.. Most of the time if we leave it then it will never be address it... 

Also you might need to update the flags here in website 
https://vitess.io/docs/16.0/reference/programs/vtctld/
In general we should update changes related to any program listed here
https://vitess.io/docs/16.0/reference/programs